docs/src/pages/components/date-picker/date-picker.md

```diff
@@ -0,0 +1,105 @@
+---
+title: React Date Picker component
+components: DatePicker, PickersDay
+githubLabel: 'component: DatePicker'
+packageName: '@material-ui/lab'
+materialDesign: https://material.io/components/date-pickers
+---
+
+# Date Picker
+
+<p class="description">Date pickers let the user select a date.</p>
+
+Date pickers let the user select a date. Date pickers are displayed with:
+
+- Dialogs on mobile
+- Text field dropdowns on desktop
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+## Requirements
+
+This component relies on the date management library of your choice. We support [date-fns](https://date-fns.org/), [luxon](https://moment.github.io/luxon/), [dayjs](https://github.com/iamkun/dayjs), [moment](https://momentjs.com/) and any other library via a public `dateAdapter` interface.
+
+Please install any of these libraries and set up the right date engine by wrapping your root (or the highest level you wish the pickers to be available) with `LocalizationProvider`:
+
+```jsx
+// or @material-ui/lab/dateAdapter/{dayjs,luxon,moment} or any valid date-io adapter
+import DateFnsAdapter from '@material-ui/lab/dateAdapter/date-fns';
+import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
+
+function App() {
+  return (
+    <LocalizationProvider dateAdapter={DateFnsAdapter}>
+      ...
+    </LocalizationProvider>
+  );
+}
+```
+
+## Basic usage
+
+The date picker will be rendered to modal dialog on mobile and textfield with popover on desktop.
+
+{{"demo": "pages/components/date-picker/BasicDatePicker.js"}}
+
+## Responsiveness
+
+The date picker component is designed and optimized for the device it runs on.
+
+- The "Mobile" version works best for touch devices and small screens.
+- The "Desktop" version works best for mouse devices and large screens.
+
+By default, the `DatePicker` component uses a `@media (pointer: fine)` media query to determine which version to use.
+This can be customized with the `desktopModeMediaQuery` prop.
+
+{{"demo": "pages/components/date-picker/ResponsiveDatePickers.js"}}
+
+## Localization
+
+Use `LocalizationProvider` to change the date-engine locale that is used to render the date picker. Here is an example of changing the locale for the `date-fns` adapter:
+
+{{"demo": "pages/components/date-picker/LocalizedDatePicker.js"}}
+
+## Views playground
+
+It is possible to combine `year`, `month`, and `date` selection views. Views will appear in the order they passed to the `views` array.
+
+{{"demo": "pages/components/date-picker/ViewsDatePicker.js"}}
+
+## Static mode
+
+It is possible to render any picker without the modal/popover and text field. This can be helpful to build custom popover/modal containers.
```

Comment:
```suggestion
It's possible to render any picker without the modal/popover and text field. This can be helpful when building custom popover/modal containers.
```